api/blueprints/drbCollection.py

```diff
@@ -379,11 +379,9 @@ def collectionList():
     )
 
     for collection in collections:
-        uuid = collection.uuid
+        path = '/collection/{}'.format(collection.uuid)
 
-        path = '/collection/{}'.format(uuid)
-
-        group = constructOPDSFeed(uuid, dbClient, perPage=5, path=path)
+        group = constructOPDSFeed(collection, dbClient, perPage=5, path=path)
```

Comment:
Shouldn't the method call be `_constructOPDSFeedForCollection` since we're passing collection as the parameter and not the uuid.

Reply:
Thanks for the catch!